src/components/UI/common/PriorityProjectsChart.tsx

```diff
@@ -0,0 +1,107 @@
+import { FC } from 'react';
+import {
+  Box,
+  ChakraProps,
+  Flex,
+  FlexProps,
+  Link,
+  ListItem,
+  Text,
+  UnorderedList
+} from '@chakra-ui/react';
+
+import { StepArrowIcon } from '../icons';
+import { ESP_WISHLIST_URL } from '../../../constants';
+
+function Arrow(props: FlexProps) {
+  return (
+    <Flex
+      direction={{ base: 'column-reverse', lg: 'row' }}
+      justifyContent='space-between'
+      alignItems='center'
+      {...props}
+    >
+      <Box
+        w={{ base: '5px', lg: 'full' }}
+        h={{ base: 'full', lg: '5px' }}
+        mr={{ base: 0, lg: '-5px' }}
```

Comment:
Soft suggestion to use direction-independent terms (`margin-inline-end`, or Chakra's `me`, instead of `margin-right`)
```suggestion
        me={{ base: 0, lg: '-5px' }}
```
...just in case this is ever translated, will help the transition.

Reply:
Good point. I need to start incorporating these props more....haha